wizeline/wize-q-front
# Feat - WIP 305 - Assign answer

#### What does this PR do?
This feature adds the option for admins to assign a question to an admin by sending them an email

#### Where should the reviewer start?
Start with src/reducers/index.js and src/reducers/adminsReducer.js, then check the home callbacks. This sets up the reducers for the getadmins call. After that check the ListQuestions and AssignAnswerModal for the UI.

#### How should this be manually tested?

Login as admin
Click on assign on any question
Select admin to assign
Submit the form and check email if you are the assigned admin

#### What are the relevant tickets?
[Ability to assign an answer request to another admin](https://wizeline.atlassian.net/browse/WIP-305)

#### Screenshots
![image](https://user-images.githubusercontent.com/29713355/27934068-a7dfb590-6269-11e7-8377-1e26631b13ba.png)
![image](https://user-images.githubusercontent.com/29713355/27970197-8792d6dc-6313-11e7-84fc-e8bddb7fe3d8.png)
